binary_search/binarysearch.cpp

```diff
@@ -0,0 +1,42 @@
+#include<iostream>
+
+using namespace std;
+
+int binary_search_iterative(int array[], int start, int end, int find)
+{
```

Comment:
The indentation is not according to standard cpp guidelines.
The opening curly brace always starts in same line.
`int name_of_func(...) {` Same changes in rest of the file.